lib/Controller/Display.php

```diff
@@ -662,20 +663,20 @@ function grid(Request $request, Response $response)
             }
 
             // Delete
-            if ($this->getUser()->checkDeleteable($display)) {
-                $display->buttons[] = array(
+            if (Environment::isDevMode() && $this->getUser()->checkDeleteable($display)) {
```

Comment:
this will make it so the delete button is only available in devMode, is that correct?

Reply:
No, not correct! I will fix it, thanks.